vendor/opam/configure.ac

```diff
@@ -0,0 +1,494 @@
+dnl The line below must be formatted AC_INIT(opam,VERSION) with no extra spaces
```

Comment:
Since we now have [ocaml-dune/opam](https://github.com/ocaml-dune/opam) I've put the changes into the `vendored` branch, so it can be (hopefully easily) rebased in the future and basically precomputed this rule, marking it `(mode fallback)`.

Thus I both dropped `configure.ac` and `get_version.ml` from being vendored.